addons/crm/tests/test_res_partner.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import TransactionCase
+
+
+class TestResPartner(TransactionCase):
+
+    def test_meeting_count(self):
```

Comment:
Could you use a real user to skip using root/admin ? Normally in 14 you have a common file holding some test users (either here, either in sales_team). Otherwise just create a salesman.